lib/linux_kernel/kthread.c

```diff
@@ -199,6 +199,7 @@ M0_INTERNAL int m0_thread_confine(struct m0_thread *q,
 {
 	int                 result = 0;
 	size_t              idx;
+	ssize_t             idx1 = -1;
```

Comment:
[* suggestion *] Use `m0_bitmap_set_nr()` instead of an additional variable here and in userspace version.

Reply:
Sorry, I'm not sure how to use it here.

Reply:
Ok, I used in M0_PRE() check instead of M0_ASSERT(). But we still need idx1 variable to figure out the 1st cpu id set in the bitmap.